src/components/layout/Logo.js

```diff
@@ -0,0 +1,13 @@
+import LogoOne from "../../img/logo/LogoOne.png";
+import LogoTwo from "../../img/logo/LogoTwo.png";
+
+function Logo() {
+  return (
+    <div className="header__logo">
+      <img className="header__logo-one" src={LogoOne} alt="logo" />
```

Comment:
Atrybut alt tutaj mógłbybyć pustym stringiem ("") ze względu na kwestie accessibility. 

Cytując internety: "If an image is used simply for decoration and provides no substantive meaning to the page, the alt attribute can be set to the empty string. alt=”” This tells screen readers to skip this image."

Reply:
Mógłby, ale czy tak naprawdę przeszkadza?

Reply:
Zeby wiedział że jest

Reply:
Cóż, najwyraźniej są różne szkoły :) Jedni mówią żeby nie dawać, a inni, żeby dawać. To niech już jest :)